code_schemes/yes_no.json

```diff
@@ -0,0 +1,91 @@
+{
+  "Version": "0.0.0.1",
+  "Name": "Yes_No Internet/Water Filter Working?",
```

Comment:
I think we should have two schemes here (one for internet, one for water), with different code ids for yes/no in each. This means we can't confuse the schemes.